src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/fields/Input.test.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react';
+import {shallow} from 'enzyme';
+import Input from '../../fields/Input';
+import InputComponent from '../../../../components/Input';
+
+test('Pass error correctly to Input component', () => {
+    const error = {keyword: 'minLength', parameters: {}};
+
+    const inputInvalid = shallow(
+        <Input
+            onChange={jest.fn()}
+            onFinish={jest.fn()}
+            value={'xyz'}
+            error={error}
+        />
+    );
+
+    expect(inputInvalid.find(InputComponent).prop('valid')).toBe(false);
+
+    const inputValid = shallow(
```

Comment:
I would put everything starting from here into a separate test. Smaller tests are easier to maintain when refactoring, and there is no dependency on the first part of the test.